src/lang/english.txt

```diff
@@ -2841,6 +2841,8 @@ STR_SAVELOAD_DETAIL_GRFSTATUS                                   :{SILVER}NewGRF:
 STR_SAVELOAD_FILTER_TITLE                                       :{BLACK}Filter string:
 STR_SAVELOAD_OVERWRITE_TITLE                                    :{WHITE}Overwrite File
 STR_SAVELOAD_OVERWRITE_WARNING                                  :{YELLOW}Are you sure you want to overwrite the existing file?
+STR_SAVELOAD_DIRECTORY                                          :{RAW_STRING} (Directory)
+STR_SAVELOAD_PARENT_DIRECTORY                                   :.. (Parent directory)
```

Comment:
Should the `..` be part of the string? I'm guessing no, because for the plain `..` on the line above it isn't localized (there's nothing to localize), so I don't think it needs to be made localized for this string either.

Reply:
[Previous version](https://github.com/OpenTTD/OpenTTD/commit/b71809ae5aa16df3754a4702bb8fea66a162215a) used `{RAW_STRING} (Parent directory)` and a constant parameter.

Reply:
Do lang files handle inline comments? Might be worth adding the justification, for translator information